src/service/resources-impl.js

```diff
@@ -445,7 +445,31 @@ export class ResourcesImpl {
 
     if (this.intersectionObserver_) {
       // The observer callback will schedule a pass to process this element.
-      this.intersectionObserver_.observe(element);
+      // DO NOT SUBMIT: a quick emulation of the FIE case. For debugging only.
```

Comment:
DO NOT SUBMIT: debugging only.

Reply:
im confused, what is this for?

Reply:
You can just ignore. But basically a way to trigger the mode w/o a viewer.

Reply:
Removed.